ts/features/messages/screens/MessageDetailsScreen.tsx

```diff
@@ -105,8 +101,8 @@ export const MessageDetailsScreen = () => {
   }
 
   return (
-    <SafeAreaView edges={["bottom"]} style={IOStyles.flex}>
-      <ScrollView>
+    <ScrollView contentContainerStyle={{ flexGrow: 1 }}>
```

Comment:
Should we move the inline style into its dedicated section, at the beginning of the file?